### DISCUSSION TITLE
Styling for `<abbr></abbr>` tags

### DISCUSSION THREAD

**Discussion by Chrissdroid** [withastro/docs#589] [Ideas]
improve the `<abbr></abbr>` tag styling for something more appealing to the eyes

**Comment by sarah11918** [comment]
(Can you believe on Apr 4 we were at #299???)

What are we still thinking about this? @Chrissdroid ?

**Comment by Chrissdroid** [comment]
( I'm in the "deny" mental stage for the moment, lol )

It could still be useful (especially in the CLI and CMS section which have their own equivalent in their language) and can improve the "experience" to indicate more data on hover of a word or sentence to add additional information

Adding default styling for this could be the minimum without touching any markdown plugins to be able to use this natively ?

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

This no longer seems to be a viable discussion since... I don't think we have any `<abbr />` tags in docs, and Starlight might have default styling for them.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
